### PR TITLE
SpatiaLite to SVG display format

### PR DESCRIPTION
**Pull request by sandman7920**

### CODE REVIEW

**Comment by justinclift** [comment]
Cool.  When we merge this, lets do:

* A blog post with at least an example + screenshot
* Make sure the nightly builds have been run, so we can point people at downloads that have the functionality for this in it

Pretty sure it'll be a useful feature for people that do GIS stuff. :smile:

**Review by mgrojo — CHANGES_REQUESTED** [review]
Nice work! Please check the duplicated line, which seems unintended, but otherwise, I think it can be merged.

File: src/ImageViewer.ui [hunk 1]
```diff
@@ -132,6 +135,7 @@
  </widget>
  <resources>
   <include location="icons/icons.qrc"/>
+  <include location="icons/icons.qrc"/>
```
[the remark this answers is not in the document]
Comment (sandman7920) [reply src/ImageViewer.ui:138]: > 
> 
> Is this intended?

Uhh no, somehow Qt Creator added this automatically

[the remark this answers is not in the document]
Comment (mgrojo) [reply src/ImageViewer.ui:138]: Sometimes Creator makes funny things with UI files. It's a good idea to review them for any undesired change.

**Comment by justinclift** [comment]
Ahhh, cool.  Didn't realise this had been merged.

We should probably do some promo of this via the website or something.

@sandman7920 Do you have an example database with SVG data in it that I can load and take some screenshots of?  Hmmm, unless you're ok to do the screenshots, and I'll throw them on the website in a blog post.

Oh, if you want to write blog post stuff, you're definitely welcome to.  No-one else but me seems to, and I'd like to kind of spread that around a bit. :wink:

**Comment by sandman7920** [comment]
@justinclift I have posted in the developer's team, I don't know if you read the post.

**Comment by justinclift** [comment]
Sorry, yeah.  It's just been a very busy few weeks.  I'll get around to it in the near-ish future. :smile:

As a reminder, everyone in the team has write access to the blog.  So, everyone can write blog posts and they'll automatically appear on the website (it checks and updates every 10 mins). :grin:

**Pull request merged**